Sources/Widgets/Widgets3D/ResliceCursorWidget/example/index.js

```diff
@@ -148,14 +148,14 @@ for (let i = 0; i < 3; i++) {
 // Load image
 // ----------------------------------------------------------------------------
 
-function updateReslice(viewtype, reslice, actor, renderer) {
+function updateReslice(viewtype, reslice, actor, renderer, renderwindow) {
```

Comment:
you can get renderwindow from renderer, no ?

Reply:
Yes, didn't see it!